src/libmw/include/mw/mmr/MMRUtil.h

```diff
@@ -4,10 +4,13 @@
 #include <mw/mmr/Index.h>
 #include <mw/mmr/LeafIndex.h>
 
+class ILeafSet;
```

Comment:
Is this forward declaration necessary? Can't see a reference to it in this file.